### ISSUE TITLE
"Welcome" page and "Getting Started" page don't open

### ISSUE THREAD

**Issue by austinbyersking** [microsoft/vscode#127206]
Issue Type: <b>Bug</b>

When I open VSCode it use to open getting started page however now it won't open. I get error below:

Unable to open '': Unable to resolve resource walkThrough://vscode_getting_started_page.

I tried this with welcome page and get the error below:

Unable to open 'vs_code_welcome_page': Unable to resolve resource walkThrough:/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/contrib/welcome/page/browser/vs_code_welcome_page?%7B%22moduleId%22%3A%22vs%2Fworkbench%2Fcontrib%2Fwelcome%2Fpage%2Fbrowser%2Fvs_code_welcome_page%22%7D.

I only get this problem with these two pages. 

VS Code version: Code 1.57.1 (Universal) (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:28:32.912Z)
OS version: Darwin x64 20.5.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8257U CPU @ 1.40GHz (8 x 1400)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|8.00GB (0.01GB free)|
|Process Argv|--crash-reporter-id a82621eb-9607-4b97-8cc2-6e097875e512|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30311712
vspre833:30321513
pythonptprofiler:30281270
vshan820:30294714
vscoreces:30322571
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255:30323308
vscorehov:30309549
vscod805:30301674
vscextlang:30322306
binariesv615:30325510

```

</details>

**Comment by JacksonKearl** [comment]
@austinbyersking do you by change have `editorAssociations: {'*': 'default'}` set anywhere in your settings? If you remove that it should work.

cc @lramos15 can we forcibly exclude `walkThrough` schemes from override?

**Comment by lramos15** [comment]
> cc @lramos15 can we forcibly exclude `walkThrough` schemes from override?

It already is excluded in insiders so this will be fixed in 1.58
